grammar.js

```diff
@@ -197,6 +197,13 @@ module.exports = grammar({
     keyword_numeric: _ => make_keyword("numeric"),
     keyword_real: _ => choice(make_keyword("real"),make_keyword("float4")),
     keyword_float: _ => make_keyword("float"),
+    unsigned_integer: _ => seq(
+        make_keyword("unsigned"),
+        choice(
+            make_keyword("integer"),
+            make_keyword("int")
+        )
```

Comment:
We already have a `keyword_int` to represent the choice of possible values. Could we use that here instead? Also, it might be good to have `unsigned` be a named node (e.g. `$.keyword_unsigned: _ => make_keyword("unsigned")`) 

```suggestion
        make_keyword("unsigned"),
        $.keyword_int,
```